test/JDBC/upgrade/latest/schedule

```diff
@@ -189,6 +189,7 @@ bitwise_not-operator
 case_insensitive_collation
 cast_numeric_types_to_datetime
 cast_numeric_types_to_smalldatetime
+col_name
```

Comment:
Nitpick:- would be better to have naming similar to object_id something like `BABEL_COL_NAME`